trace_point.rb

```diff
@@ -153,7 +153,7 @@ def self.allow_reentry
 
   # call-seq:
   #    trace.enable(target: nil, target_line: nil, target_thread: nil)    -> true or false
-  #    trace.enable(target: nil, target_line: nil, target_thread: nil) { block }  -> obj
+  #    trace.enable(target: nil, target_line: nil, target_thread: :default) { block }  -> obj
```

Comment:
How about to write it as `Thread.current`?

Reply:
The problem with that is that it gives the impression that the default is always the current thread, when the behavior is that the default is only the current thread if `:target` and `:target_line` are not provided.

Reply:
I see. Thank you!